package.json

```diff
@@ -19,6 +18,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test:coverage": "CI=true npm test -- --env=jsdom --coverage",
```

Comment:
I love that we added this and are getting such high testing coverage.

Reply:
Yeah, this is awesome.  Maybe we should look into getting this command to run when we run 'npm test'